cli/src/main/java/org/opentosca/toscana/cli/commands/ToscanaHelp.java

```diff
@@ -0,0 +1,119 @@
+package org.opentosca.toscana.cli.commands;
+
+import org.opentosca.toscana.cli.commands.csar.CsarDelete;
+import org.opentosca.toscana.cli.commands.csar.CsarInfo;
+import org.opentosca.toscana.cli.commands.csar.CsarList;
+import org.opentosca.toscana.cli.commands.csar.CsarUpload;
+import org.opentosca.toscana.cli.commands.csar.ToscanaCsar;
+import org.opentosca.toscana.cli.commands.platform.PlatformInfo;
+import org.opentosca.toscana.cli.commands.platform.PlatformList;
+import org.opentosca.toscana.cli.commands.platform.PlatformStatus;
+import org.opentosca.toscana.cli.commands.platform.ToscanaPlatform;
+import org.opentosca.toscana.cli.commands.transformation.ToscanaTransformation;
+import org.opentosca.toscana.cli.commands.transformation.TransformationDelete;
+import org.opentosca.toscana.cli.commands.transformation.TransformationDownload;
+import org.opentosca.toscana.cli.commands.transformation.TransformationInfo;
+import org.opentosca.toscana.cli.commands.transformation.TransformationList;
+import org.opentosca.toscana.cli.commands.transformation.TransformationStart;
+import org.opentosca.toscana.cli.commands.transformation.TransformationStatus;
+import org.opentosca.toscana.cli.commands.transformation.TransformationStop;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import static picocli.CommandLine.usage;
+
+@Command(name = "help",
+    header = {"Help Page", ""},
+    customSynopsis = "@|bold toscana help|@ [@|yellow <command>|@] [@|yellow <subcommand>|@]",
+    optionListHeading = "%nOptions:%n",
+    descriptionHeading = "%nAvailable TOSCAna CLI commands and subcommands:%n",
+    description = {"csar%n   delete%n   info%n   list%n   upload%nhelp%nplatform%n   info",
+        "   list%n   status%nstatus%ntransformation%n   delete%n   download%n   info",
+        "   list%n   start%n   status%n   stop%n"})
+public class ToscanaHelp extends AbstractCommands implements Runnable {
+
+    @Parameters(arity = "0..1", paramLabel = "command", description = "Shows help page for every command", hidden = true)
+    public String[] help;
+
+    /**
+     * ToscanaHelp shows a help page for every available command
+     */
+    public ToscanaHelp() {
+    }
+
+    @Override
+    public void run() {
+        String h = "";
+
+        if (help != null) {
+            if (help.length > 1) {
+                h += help[0] + " " + help[1];
+            } else {
+                h += help[0];
+            }
+
+            switch (h) {
+                case "csar":
+                    usage(new ToscanaCsar(), System.err);
```

Comment:
I Am assuming you append the usage to Syserr. I think sysout (system.out) ist the better output for this

Reply:
I just used it like in the Documentation

Reply:
Write expected behaviour to `stdout` and only error messages to `stderr`. In the upper case, write to `stdout`.